cmd/mutating.go

```diff
@@ -0,0 +1,118 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+
+	"github.com/olekukonko/tablewriter"
+	"github.com/kubernetes-sigs/pspmigrator"
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var MutatingCmd = &cobra.Command{
+	Use:   "mutating",
+	Short: "Check if pods or PSP objects are mutating",
+}
+
+func initMutating() {
```

Comment:
the order got messed up when there is multiple init functions in a single package. So that's why I'm using a single init() in the package and then calling initMutating from the main init() in the package which is in cmd/root.go. There might be better ways to handle this but it fixed the issue I had with order.